homework-submission/week4/src/components/week1/App.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Body = styled.body`
+  background: white;
+  color: black;
+  height: 100vh;
+  margin: 0;
+  padding: 2rem;
+  justify-content: center;
+  align-items: center;
+`;
+
+const H1 = styled.h1`
+  margin-top: 2rem;
+  font-weight: 400;
+`;
+
+class Header extends React.Component {
+  render() {
+    return <H1>{this.props.title}</H1>;
+  }
+}
+
+class ListItem extends React.Component {
+  render() {
+    return (
+      <div>
+        <ul>
+          <li>
+            {this.props.description}, {this.props.deadline}
+          </li>
+        </ul>
+      </div>
+    );
+  }
+}
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [
+        {
+          id: 1,
+          description: 'Get out of bed',
+          deadline: '2017-09-11',
+          done: true,
+        },
+        {
+          id: 2,
+          description: 'Brush teeth',
+          deadline: '2017-09-10',
+          done: false,
+        },
+        {
+          id: 3,
+          description: 'Eat breakfast',
+          deadline: '2017-09-09',
+          done: false,
+        },
+      ],
+    };
+  }
+
+  render() {
+    return (
+      <Body>
```

Comment:
You can't render a `body` inside the `<div id="root">` because that div is already inside the main `<body>`. Rendering this one as a `div` fixes it. Make sure to look at the dev tools, there are some handy warnings there.

Reply:
Done :pray: